Backend/package.json

```diff
@@ -23,8 +24,12 @@
     "express": "^4.17.1",
     "google-auth-library": "^6.0.0",
     "graphql": "^14.0.0",
+    "graphql-passport": "^0.6.3",
     "mongodb": "^3.5.7",
-    "ncp": "^2.0.0"
+    "ncp": "^2.0.0",
+    "passport": "^0.4.1",
+    "passport-google-oauth": "^2.0.0",
+    "passport-google-oauth20": "^2.0.0"
```

Comment:
Are both used? :)

Reply:
Not sure, typescript was angry so i tried a lot to make it calm down